clutchtimevite/src/components/feed/index.jsx

```diff
@@ -346,13 +349,28 @@ const Feed = () => {
                   ...leagueNews.map((news) => ({
                     ...news,
                     league,
+                    id: `news_${newsId++}`, //assigns unique ID to each news item
```

Comment:
you might consider a different method of generating a unique id such as https://stackoverflow.com/questions/105034/how-do-i-create-a-guid-uuid